surfsense_backend/app/tasks/connectors_indexing_tasks.py

```
@@ -1651,3 +1652,241 @@ async def index_discord_messages(
+                f"Failed to find Google Drive connector {connector_id}",
+                error_msg,
+                {"error_type": "ConnectorNotFound"}
+            )
+            logger.error(error_msg)
+            return 0, error_msg
+        
+        if connector.connector_type != SearchSourceConnectorType.GOOGLE_DRIVE_CONNECTOR:
+            error_msg = f"Connector {connector_id} is not a Google Drive connector"
+            await task_logger.log_task_failure(
+                log_entry,
+                f"Invalid connector type for Google Drive indexing",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)